testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -86,8 +91,9 @@ def test_replication_configuration_valid_values(params_from_base_test_setup, num
     # Configure replication with push_pull
     replicator = Replication(base_url)
     sg_client.create_user(sg_admin_url, sg_db, username, password, channels=channels_sg)
-    session, replicator_authenticator, repl = replicator.create_session_configure_replicate(
-        base_url, sg_admin_url, sg_db, username, password, channels_sg, sg_client, cbl_db, sg_blip_url, continuous=continuous, replication_type="push_pull")
+    session, _, repl = replicator.create_session_configure_replicate(
```

Comment:
what does the _, represent for?

Reply:
it's a placeholder, You assign a variable like this if you are not going to use the value.